osc-dm-product-srv/src/server.py

```diff
@@ -226,6 +226,60 @@ async def dataproducts_uuid_metrics_get(uuid: str):
     return response
 
 
+
+@app.post(ENDPOINT_PREFIX + "/products/generate")
+async def dataproducts_generate_product_post(product: models.ProductGen):
+
+    """
+    Get metrics information
+    """
+    logger.info(f"called generate endpoint with: {product}")
+    output_dir = "./dataproducts/" + product.outputDir
+    from utl.product_generator import create_product_file
+    out = create_product_file(
+        output_dir,
+        product.fileName,
+        product.namespace,
+        product.name,
+        product.tags,
+        product.description,
+        product.url,
+        product.vendor,
+        product.model)
+    from utl.uuid_generator import generate_product_uuid
+    out = generate_product_uuid(product.outputDir)
+    return out
+
+
+@app.post(ENDPOINT_PREFIX + "/products/generate/artifact")
+# async def dataproducts_generate_product_post(artifact: models.ArtifactGen):
+async def dataproducts_generate_product_artifact_post(artifact: models.ArtifactGen):
+
+    """
+    Get metrics information
```

Comment:
copy-paste issue results in wrong description

Reply:
updated documentation